src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -25,6 +27,7 @@
 from ceph.deployment.service_spec import ServiceSpec, NFSServiceSpec, RGWSpec
 from ceph.utils import datetime_now
 from ceph.deployment.drive_selection.matchers import SizeMatcher
+from nfs.cluster import create_ganesha_pool
```

Comment:
According to [PEP 8 style guide](https://www.python.org/dev/peps/pep-0008/#imports), imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports

```suggestion
from nfs.cluster import create_ganesha_pool
from nfs.module import Module
from nfs.export import NFSRados
```